src/main/kotlin/uk/gov/dluhc/printapi/messaging/ProcessPrintRequestBatchMessageListener.kt

```diff
@@ -24,7 +24,10 @@ class ProcessPrintRequestBatchMessageListener(
 
             val certificates = processPrintBatchService.processBatch(batchId, printRequestCount)
             certificates.forEach {
-                statisticsUpdateService.triggerVoterCardStatisticsUpdate(it.sourceReference!!)
+                if (payload.isFromApplicationsApi == true)
```

Comment:
I don't think we need the == true here, can just have `if(payload.isFromApplicationsApi)`

Reply:
But it is Boolean?, so cany just check if(payload.isFromApplicationsApi)

Reply:
Ah yeah you're right, we do want this given it can be null